packages/cli/lib/model-discoverer.js

```diff
@@ -0,0 +1,105 @@
+const debug = require('./debug')('model-discoverer');
+const fs = require('fs');
+const path = require('path');
+
+/**
+ * Given a datasource and discovery options,
+ * return a list of objects {table: 'foo', schema: 'bar}
+ */
+async function discoverModelNames(ds, options) {
+  if (!ds.connected) {
+    await new Promise(resolve => {
+      ds.on('connected', resolve);
+    });
+  }
+  return await ds.discoverModelDefinitions(options);
+}
+
+/**
+ * Returns the entire model definition for a model
+ */
+async function discoverSingleModel(ds, options, name) {
```

Comment:
```suggestion
async function discoverSingleModel(ds, modelName, options) {
```

`options` is usually the last argument. I find the argument name `name` as too generic, let's use `modelName` to make it obvious what kind of a name we are expecting.

Reply:
done